go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/cheekybits/genny v1.0.0
 	github.com/golang/protobuf v1.3.2
 	github.com/google/go-cmp v0.3.1
-	github.com/hashicorp/go-hclog v0.0.0-20180709165350-ff2cf002a8dd
+	github.com/hashicorp/go-hclog v0.12.1
```

Comment:
We don't want to change this since github.com/hashicorp/go-plugin depends on github.com/hashicorp/go-hclog v0.0.0-20180709165350-ff2cf002a8dd

Reply:
👍 -- any way to enforce that in the plugins that use the SDK?  I updated this because the existing plugins are using v0.12.1 (likely just because... not because they need it or anything)